src/Simplex_tree/include/gudhi/Simplex_tree.h

```diff
@@ -2246,6 +2277,18 @@ class Simplex_tree {
   /** \brief Upper bound on the dimension of the simplicial complex.*/
   int dimension_;
   bool dimension_to_be_lowered_ = false;
+
+//MULTIPERS STUFF
+public: 
+  void set_number_of_parameters(int num){
+		number_of_parameters_ = num;
```

Comment:
This is a comment for all your changes, but we are using 2 spaces as indentation for the c++ part, cf. https://github.com/GUDHI/gudhi-devel/blob/master/.github/code_conventions.md#code-style